selftests/unit/test_utils_params.py

```diff
@@ -86,6 +86,96 @@ def testGetItemMissing(self):
     def testGetItem(self):
         self.assertEqual(self.params['image_size'], "10G")
 
+    def testCfgEnabled(self):
+        self.params["something"] = "no"
+        self.params["foobar"] = "yes"
+
+        self.assertEqual(False, params.is_cfg_enabled('something'))
+        self.assertEqual(False, params.is_cfg_enabled('notgiven'))
+        self.assertEqual(True, params.is_cfg_enabled('foobar'))
+
+    def testMultiParamNone(self):
+        self.assertEqual(0, len(self.params.get_multi_param("something")))
+
+    def testMultiParamOnlyOne(self):
+        self.params['something'] = 'foobar'
+
+        found = self.params.get_multi_param("something")
+        self.assertEqual(1, len(found))
+        self.assertEqual('foobar', found[0])
+
+    def testMultiParamMany(self):
+        self.params['something'] = 'foobar'
+        self.params['something2'] = 'foobar2'
+        self.params['something3'] = 'foobar3'
+
+        found = self.params.get_multi_param("something")
+        self.assertEqual(3, len(found))
+        self.assertEqual('foobar', found[0])
+        self.assertEqual('foobar2', found[1])
+        self.assertEqual('foobar3', found[2])
+
+    def testIsObjectSpecific(self):
+        self.assertTrue(is_object_specific("genie_vm1", ["vm1", "vm2"]))
+        self.assertFalse(is_object_specific("genie_vm1", ["nic1", "nic2"]))
+        self.assertTrue(is_object_specific("god_vm2", ["vm1", "vm2"]))
+        self.assertFalse(is_object_specific("god_vm2", ["nic1", "nic2"]))
+        self.assertFalse(is_object_specific("genie", ["vm1", "vm2"]))
+        self.assertFalse(is_object_specific("god", ["vm1", "vm2"]))
+        self.assertFalse(is_object_specific("wizard", ["vm1", "vm2"]))
+        self.assertFalse(is_object_specific("wizard", ["wizard"]))
+
+    def testObjectParams(self):
+        vm_params = utils_params.Params({"name_vm1": "josh", "name_vm2": "jean", "name": "jarjar", "surname": "jura"})
+        params = object_params(vm_params, "vm1", ["vm1", "vm2"])
```

Comment:
on many places you use `utils_params` methods without using `utils_params` module resulting in failures... (I mean object_params`, `is_object_specific` and such are not directly imported, but imported with `utils_params`...)

Reply:
Yes, old version again - however, you did get one or two instances more - thanks! I wish I could detect them but the unit tests are failing before reaching this stage.